src/components/templates/ProductTemplate/ProductTemplateLoading/ProductTemplateLoading.md

```diff
@@ -0,0 +1,5 @@
+### Summary
+
+`<ProductTemplateLoading>`
+
+// TODO
```

Comment:
I don't understand the purpose of this component

Reply:
It was a WIP - not sure its really needed - but it was originally there to represent a loading state for the template - which arguably we should have but don't at the moment.

Reply:
At the moment it doesn't do anything so can kill it

Reply:
There are separate designs for loading state, there is a full page version and modal version